### ISSUE TITLE
Streamlining installation using a conda environment file

### ISSUE THREAD

**Issue by multimeric** [Rappsilber-Laboratory/AlphaLink2#10]
Hi, I was recently trying to set up this pipeline on our HPC, and it occurred to me that we could streamline all the `conda install`, `pip install` and `python setup.py install` commands by exporting a conda environment definition file, so that users would only need to download it, then run `conda env create -f=/path/to/environment.yml` and everything should work.

Would you be interested in me contributing something like that?

**Comment by lhatsk** [comment]
Hi,
Yes, that would great! I guess the only manual step that will remain is to copy stereo_chemical_props.txt?

Thanks!

**Comment by multimeric** [comment]
Yeah so conda won't be able to handle data files like that, but I wonder if that file could be obtained within the AlphaLink2 code, either during setup or at runtime, so users don't have to manually install it?

**Comment by lhatsk** [comment]
Hm, where would you include it? E.g., download/ copy at the beginning of run_alphalink.sh if it doesn't exist yet?

**Comment by multimeric** [comment]
I don't know if it's subject to any license issues that prevent it, but what I would do is include it in the package as package data: https://setuptools.pypa.io/en/latest/userguide/datafiles.html. To do that all you need to do is commit it to the git repository and ensure the project metadata is configured correctly. It's only ~ 9 kilobytes so I doubt anyone would really mind.

**Comment by lhatsk** [comment]
I don't know about the licensing, my guess is there is some issue, otherwise Deepmind would have also packaged it. I'll check it out. I guess just doing a quick check and downloading if necessary isn't too bad.
